### ISSUE TITLE
IE8 falls into infinit loop in getViewportWidthInCssPixels()

### ISSUE THREAD

**Issue by tokkonopapa** [tubalmartin/riloadr#23]
In IE8, the variable `widths` looks like `[null, 600, 800, 600]` and will be `[600, 800, 600]` by `widths.splice(i, 1);`. 
But the variable `l` is still `4`. So `i` will refer to the out of range.

The below code can fix this issue.

``` javascript
function getViewportWidthInCssPixels() {
...
    for (; i < l; i++) {
        // If not a number remove it
        if (isNaN(widths[i])) {
            widths.splice(i, 1);
            i--;
            l--; // need decrement
        }
    }
...
}
```

**Comment by tubalmartin** [comment]
Thank you for spotting that bug. It's already fixed in 1.5.3. Could you give it a try?

**Comment by tokkonopapa** [comment]
I tested 1.5.3 using IE8 and found everything is fine!
Thanks for quick update!!
